server/server_helpers/channel_mapper.js

```diff
@@ -0,0 +1,36 @@
+
+module.exports = {
+  "1": "Film & Animation",
+  "2":"Autos & Vehicles",
```

Comment:
this object is missing some keys like 3, 4, .... It is doesn't have 42 items so better change this object and adjust the fake data generator